File: DemoScrollViewPageControl/DemoScrollViewPageControl/CollectionViewCell.m
//
//  CollectionViewCell.m
//  DemoScrollViewPageControl
//
//  Created by Chris Hu on 16/5/9.
//  Copyright © 2016年 icetime17. All rights reserved.
//

#import "CollectionViewCell.h"

@implementation CollectionViewCell

- (void)awakeFromNib {
    [super awakeFromNib];
    // Initialization code
    
    [self bringSubviewToFront:self.btn];
}

- (IBAction)actionBtn:(UIButton *)sender {
    NSLog(@"%s", __FUNCTION__);
}


@end

File: DemoHero/DemoHero/CollectionViewCell.swift
//
//  CollectionViewCell.swift
//  DemoHero
//
//  Created by Chris Hu on 17/1/5.
//  Copyright © 2017年 com.icetime17. All rights reserved.
//

import UIKit

class CollectionViewCell: UICollectionViewCell {

    @IBOutlet weak var imageView: UIImageView!
    
    @IBOutlet weak var blurView: UIVisualEffectView!
    
    override func awakeFromNib() {
        super.awakeFromNib()
        // Initialization code
    }

}
